exercises/triangle/spec/triangle_spec.cr

```diff
@@ -2,59 +2,79 @@ require "spec"
 require "../src/*"
 
 describe "Triangle" do
-  it "true if all sides are equal" do
-    Triangle.new([2, 2, 2]).equilateral?.should eq(true)
+  it "all sides are equal" do
+    Triangle.new({2.0, 2.0, 2.0}).equilateral?.should eq(true)
```

Comment:
Here is an opinion from a random bystander (me). This opinion **is not intended** to be a substitute for maintainer opinion.

I think it would be preferable if these were written as `{2, 2, 2}` instead of `{2.0, 2.0, 2.0}`, and only use floats for the specific tests that use floats. It is the approach I took in https://github.com/exercism/crystal/pull/182. It still has not been reviewed.

Reply:
Good idea! I found a way to do that, done.

Thing is, previously, the exercise didn't have any floats at all. ?

Reply:
> Thing is, previously, the exercise didn't have any floats at all. ?
That is a correct statement. Perhaps you had another related question?